### PR TITLE
dogecoin: 1.14.2 -> 1.14.3

### PR DESCRIPTION
**Pull request by centromere**
###### Motivation for this change

Upgrade to latest version.

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 121941` at 78ba154e run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package skipped due to time constraints:</summary>
  <ul>
    <li>dogecoin</li>
  </ul>
</details>
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>dogecoind</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

util-linux is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/applications/blockchains/dogecoin.nix:20:3:
```
   |
20 |   buildInputs = [ openssl db5 openssl util-linux
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/applications/blockchains/dogecoin.nix [hunk 1]
```diff
@@ -7,18 +7,17 @@
...
     rev = "v${version}";
-    sha256 = "1gw46q63mjzwvb17ck6p1bap2xpdrap08szw2kjhasa3yvd5swyy";
+    sha256 = "15yjzyvgic96pybadfk37zb032xvra0v37iphbnh15dci2fd934j";
   };
 
-  nativeBuildInputs = [ pkg-config autoreconfHook ];
-  buildInputs = [ openssl db5 openssl util-linux
-                  protobuf boost zlib miniupnpc libevent ]
+  nativeBuildInputs = [ pkg-config autoreconfHook util-linux ];
+  buildInputs = [ openssl db5 openssl protobuf boost zlib miniupnpc libevent ]
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/dogecoin.nix:20]:
```suggestion
  buildInputs = [ openssl db5 rotobuf boost zlib miniupnpc libevent ]
```

Comment (centromere) [reply pkgs/applications/blockchains/dogecoin.nix:20]: The [README](https://github.com/dogecoin/dogecoin/blob/master/doc/build-unix.md#dependencies) states that `libssl` is required.

**Comment by prusnak** [comment]
There is already a PR for dogecoin here - https://github.com/NixOS/nixpkgs/pull/114769

**Comment by ajs124** [comment]
This has conflicts + we're on a newer version already. Can we close this?

**Pull request closed without merging**